blueoil/io/file_io.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 The Blueoil Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+import os
+import tempfile
+
+import numpy as np
+from PIL import Image
+from tensorflow.io import gfile
+
+
+def _is_local_path(path: str) -> bool:
+    for p in {"gs://", "s3://", "hdfs://"}:
+        if path.startswith(p):
+            return False
+    return True
```

Comment:
You can pass a tuple to `startswith`.
https://docs.python.org/3/library/stdtypes.html#str.startswith

Reply:
Oh, thanks! I simplify checking the local path and remove this function. Please check it! 👍